src/libs/antares/study/parts/short-term-storage/cluster.cpp

```diff
@@ -57,7 +57,10 @@ bool STStorageCluster::loadFromSection(const IniFile::Section& section)
     }
 
     if (properties.name.empty())
+    {
```

Comment:
Every cluster property value seems to be checked in **Properties::validate(...)** but **properties.name**, whose value is checked here.
Why is that ?

Reply:
Used to generate the cluster ID, that will be used after to load series
Validate() will be called after loading series because it need all data to check values